Dockerfile

```diff
@@ -0,0 +1,80 @@
+# In any directory on the docker host, perform the following actions:
+#   * Copy this Dockerfile in the directory.
+#   * Create input and output directories: mkdir -p oe-core oe-core/build
+#   * Build the Docker image with the following command:
+#     docker build --no-cache --build-arg "host_uid=$(id -u)" --build-arg "host_gid=$(id -g)" \
```

Comment:
`--no-cache` for everything including the base is gonna take a while, isn't it? I see that it's necessary because the build configuration is done with `RUN` commands, but maybe we could either
- Split this into two dockerfiles run by the orchestration script I mentioned above, such that the first one is everything down to like line 72 of this file, and does not require `--no-cache`; and the second one uses the image made by the first one as a base, and does use `--no-cache`
- Move the configuration stuff and build stuff into a single shell script or makefile that gets `RUN` by the dockerfile, or more likely used as a `CMD`

It'd be great to reuse as much of the generated images as possible.